shared/js/media/media_frame.js

```diff
@@ -47,6 +47,8 @@ function MediaFrame(container, includeVideo, maxImageSize) {
   this.image = new Image();
   this.image.className = 'image-view';
   this.image.style.opacity = 0;                        // Start off hidden.
+  this.image.style.transitionProperty = 'opacity';
+  this.image.style.transitionDuration = '100ms';
```

Comment:
Put the styles in a stylesheet please. See apps/camera/style.preview-gallery.css:43.
If you think this change is good for gallery too, feel free to change gallery.css as well.